tests/helpers.nim

```diff
@@ -55,10 +55,6 @@ template checkTrackers*() =
     if tracker.isLeaked():
       checkpoint tracker.dump()
       fail()
-  # Also test the GC is not fooling with us
-  try:
-    GC_fullCollect()
-  except: discard
```

Comment:
Let's keep this to be safe

Reply:
How does it increase safety? It is used only in tests and apparently, it is not necessary. We can always add it back If we start having issues.

Reply:
Honestly I have no idea why this is here, and the git blame doesn't really explains it either
But I think it's better to keep it, it doesn't cost anything and may cause weird issue that someone already had to go through

Reply:
I have no problem leaving it, but IMHO it's better to try to make the codebase cleaner and remove code that no one can explain why it is there, than leaving it. Especially in tests where the risks are low. I would even argue that it is safer to let it crash and fix the root cause than run a full GC and swallow whatever is raised.

Reply:
I don't think GC_fullCollect can actually raise anything
Issues it might cause are more related to flaky tests etc
You can just use exceptionToAssert to be safe regarding exceptions

Reply:
I reverted it.